doc/source/user_guide/index.rst

```diff
@@ -0,0 +1,52 @@
+.. _ref_user_guide:
+
+User guide
+##########
+
+Granta MI record lists
+----------------------
+
+Record lists are collections of records, and can be used in any situation where it is useful to have
+a persistent, server-side list of records. Records are explicitly added to a list by a user, and so
+do not need to have anything in common (as would be typical for a set of search results), and do not
+necessarily need be in the same table or database. The user has complete flexibility in deciding
+which records to add to a list.
+
+The record list management capability is available to users in the following front-end tools:
+
+* The One MI Favorites app, as 'Favorites lists'
```

Comment:
The single backtick in this line and the next will format the text in italics. This might be what you want. However, if **Favorites list** is the name of a UI element in the app or product, it should be in bold text.

Reply:
I had them just as single quotes with no special formatting. I will change to `**` though for bold text